Extension/src/LanguageServer/client.ts

```diff
@@ -1240,6 +1240,7 @@ export class DefaultClient implements Client {
                 closed: () => {
                     languageClientCrashTimes.push(Date.now());
                     languageClientCrashedNeedsRestart = true;
+                    telemetry.logLanguageServerEvent("languageClientCrash", { "count": languageClientCrashTimes.length.toString() }, undefined);
```

Comment:
I think the count is maxed at 5.  So since that is lossy, let's just not add a count at all. We can just count the number of events that fire.

Reply:
That seems fine, but the count would tell us if it's a rare crash that just occurred just 1 time in a session or a repeated/severe crash that crashed 5 times in 3 minutes or not...not sure if that's important enough to track or not.